clustered/server/src/main/java/org/ehcache/clustered/server/state/MessageTracker.java

```diff
@@ -80,6 +85,17 @@ public boolean isEmpty() {
    * Remove the contiguous seen msgIds from the nonContiguousMsgIds and update highestContiguousMsgId
    */
   private void reconcile() {
+
+    // This happens when a passive is started after Active has moved on and
+    // passive starts to see msgIDs starting from a number > 0.
+    if (highestContiguousMsgId == -1L && nonContiguousMsgIds.size() > 100) {
```

Comment:
@ljacomet nope active don't track message ids because its not required. Only Passives track it, thus there nothing to sync from Active to Passive. 
Also 100 is not a magic number, I have run tests to come to that, also we can discuss that I can explain you